src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -50,8 +58,11 @@
 
     @Test
     public void getStatisticAboutGrape() {
+        try {
         workWithFile.getStatistic("grape.csv", GRAPE_RESULT_FILE);
-
+        } catch (IOException e) {
+            System.out.println("Can't write data to file");
+        }
```

Comment:
The try-catch block for `IOException` is not necessary in test methods. Letting the exception propagate will cause the test to fail and provide more detailed information about the failure.